iree/test/e2e/models/mnist_fake_weights.mlir

```diff
@@ -5,10 +5,10 @@
 // RUN: [[ $IREE_VULKAN_DISABLE == 1 ]] || (iree-run-mlir -iree-hal-target-backends=vulkan-spirv %s -function-input="1x28x28x1xf32" | IreeFileCheck %s)
 
 module attributes {tf.versions = {bad_consumers = [], min_consumer = 12 : i32, producer = 175 : i32}} {
-  flow.variable @"__iree_flow___sm_node15__model.layer-2.kernel" dense<0.5> : tensor<784x128xf32>
-  flow.variable @"__iree_flow___sm_node16__model.layer-2.bias" dense<0.1> : tensor<128xf32>
-  flow.variable @"__iree_flow___sm_node21__model.layer-3.kernel" dense<0.5> : tensor<128x10xf32>
-  flow.variable @"__iree_flow___sm_node22__model.layer-3.bias" dense<0.1> : tensor<10xf32>
+  flow.variable @"__iree_flow___sm_node15__model.layer-2.kernel" dense<0.5> : tensor<784x128xf32> attributes {sym_visibility = "private"}
+  flow.variable @"__iree_flow___sm_node16__model.layer-2.bias" dense<0.1> : tensor<128xf32> attributes {sym_visibility = "private"}
+  flow.variable @"__iree_flow___sm_node21__model.layer-3.kernel" dense<0.5> : tensor<128x10xf32> attributes {sym_visibility = "private"}
+  flow.variable @"__iree_flow___sm_node22__model.layer-3.bias" dense<0.1> : tensor<10xf32> attributes {sym_visibility = "private"}
```

Comment:
This file is forked from https://github.com/google/iree/blob/main/iree/samples/models/mnist.mlir [1], which was/is generated from TensorFlow Python. Can these attributes be set automatically somewhere in the compiler? We (read: "I") can regenerate from Colab if so. I'd generally prefer to not make direct edits to these files outside of the compiler, since that can hide missing features in our tools.


[1]: Maybe the whole https://github.com/google/iree/tree/main/iree/samples/models directory should be deleted now. We have a cloud storage bucket somewhere for larger models, right?

Reply:
The compiler already does add these, but this was exported prior to that change:
https://github.com/google/iree/blob/9228670a0e06cf52589f141f7fb171fe574089d7/integrations/tensorflow/compiler/TFSavedModelLowerGlobalTensors.cpp#L85-L86

A bucket (and a good way to put new things in it) would be fantastic for getting the larger sample models in. Still worth having a few in-repo, though.

Reply:
Okay, manual update is fine then. I'll make a note to re-export so it doesn't drift too much.